Content.Client/Chat/ChatBox.cs

```diff
@@ -7,18 +8,30 @@
 using Robust.Shared.Maths;
 using Robust.Shared.Utility;
 
+
 namespace Content.Client.Chat
 {
     public class ChatBox : PanelContainer
     {
         public delegate void TextSubmitHandler(ChatBox chatBox, string text);
 
+        public delegate void FilterPressedHandler(ChatBox chatBox, Button.ButtonEventArgs e);
```

Comment:
You should make the second parameter of type `Button.ButtonToggledEventArgs`, and probably rename the event to use `Toggled` too.

Reply:
Changed Pressed to Toggled wherever necessary.